subprojects/logging/src/main/java/org/gradle/internal/logging/slf4j/OutputEventListenerBackedLogger.java

```diff
@@ -130,7 +129,11 @@ public void trace(Marker marker, String msg, Throwable t) {
 
     private void log(LogLevel logLevel, Throwable throwable, String message) {
         OperationIdentifier buildOperationId = CurrentBuildOperationRef.instance().getId();
-        LogEvent logEvent = new LogEvent(clock.getCurrentTime(), name, logLevel, message, throwable, buildOperationId);
+        log(logLevel, throwable, message, buildOperationId);
```

Comment:
Inline?

Reply:
not sure what you mean. inlining is not working here as we have ` log(logLevel, throwable, message, buildOperationId)` with explicit build operation id used that is coming from `BuildOperationAwareLogger`

Reply:
I just meant don't save to a variable, that's all.

Reply:
fixed